.github/workflows/ci.yml

```diff
@@ -105,16 +117,14 @@ jobs:
             adb uninstall com.slack.keeper.sample || true
             adb uninstall com.slack.keeper.sample.androidTest || true
 
-      - name: (Fail-only) Bundle the build report
-        if: failure()
-        run: find . -type d -name 'reports' | zip -@ -r build-reports.zip
```

Comment:
zip doesn't exist on windows so this fails there (and the builds fail there so it was obvious!)

I ran all the commands locally (including the tricky `CI=true` in front of the validateL8 one to check what dirs matched this `find` command

There were only two, so I directly specify them in upload-artifact below, and the artifact should have the same contents now, after this change